src/operatorApp/tests/operator.test.js

```diff
@@ -180,4 +180,140 @@ describe('Operator router tests', () => {
 			.set('Authorization', `Bearer ${token}`);
 		expect(response).to.have.status(400);
 	});
+	// Testing for logged in as an operator
+	it('should login an operator user', async () => {
+		const password = await bcrypt.hash('Andela@1234', 5);
+		const newUser = User.create({
```

Comment:
hey @feyton , could you create a separate file and put in these testing dummy data?